docs/ci.yaml

```diff
@@ -0,0 +1,27 @@
+# Continuous Integration
+
+## GitHub Workflows
+
+The GitHub Workflows under [`.github/workflows/][workflows] contain jobs that
+are started when Pull-Requests are created or updated. Some of the jobs can
+build container-images for multiple architectures. Not everyone or all
+environmens wants to run the build tests for all platforms. The workflows can
+be configured to select platforms that the `docker/setup-buildx-action`
+supports.
+
+For this configuration, a new Environment should be created in the GitHub
+Settings of the repository. The Environment is expected to be called `ci`, and
+a single Secret ('normal' environment variables seem not possible) should be
+added:
+
+- `BUILD_PLATFORMS`: `linux/amd64,linux/arm64,linux/arm/v7`
```

Comment:
this is already configured or need to be configured?

Reply:
I have configured it. This PR should have built for all these architectures

Reply:
not yet, the CI is passing in less than 2 minutes.

Reply:
indeed, that is wrong!

Reply:
I just re-set the variable, and restarted the jobs... There is no difference, still only a single architecture. I tested this yesterday in my personal fork, and there it worked. Wondering what the difference is.

Reply:
Now moved `BUILD_PLATFORMS` into a regular secret, not in an environment secret... Unfortunately that does not seem to help either :thinking:

Reply:
https://github.com/nixpanic/kubernetes-csi-addons/pull/2/checks seems to do it's job as intended... There is no difference between the configuration of the two GitHub repositories as far as I can see.